src/odbIdentityProvider.js

```diff
@@ -34,15 +33,22 @@ class OdbIdentityProvider {
     return (await this.threeId.signJWT(payload, opts)).split('.')[2]
   }
 
+  static async setDidResolver(resolver) {
+    this._resolver = resolver
+  }
```

Comment:
Since this is a static function does it really have a `this` that we can use?

Reply:
`this` in a static function refers to the class/prototype itself - this is like a static class attribute. We could also use `OdbIdentityProvider._resolver`, but using `this` allows inheritance.

Reply:
ok I see, thanks.